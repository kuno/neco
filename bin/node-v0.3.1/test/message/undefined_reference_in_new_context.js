common = require("../common");
assert = common.assert

common.error('before');

var Script = process.binding('evals').Script;

// undefined reference
script = new Script('foo.bar = 5;');
script.runInNewContext();

common.error('after');
