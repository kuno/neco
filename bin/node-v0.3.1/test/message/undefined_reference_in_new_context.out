before

node.js:*
    throw e; // process.nextTick error, or 'error' event on first tick
    ^
ReferenceError: foo is not defined
    at evalmachine.<anonymous>:*
    at *test/message/undefined_reference_in_new_context.js:*
    at Module._compile (node.js:*)
    at Object..js (node.js:*)
    at Module.load (node.js:*)
    at Array.<anonymous> (node.js:*)
    at EventEmitter._tickCallback (node.js:*)
